docs/source/mirdata.rst

```diff
@@ -282,6 +282,14 @@ maestro
    :inherited-members:
 
 
+mdb\_stem\_synth
+^^^^^^^^^^^^^^^
+
+.. automodule:: mirdata.datasets.mdb_stem_synth
+   :members:
+   :inherited-members:
+
+
```

Comment:
This is not properly rendering [in the docs](https://mirdata--639.org.readthedocs.build/en/639/source/mirdata.html#mdb-stem-synth). I think you don't need the `\`, no? Also, you are missing a `^` below the title. I wonder if that would solve the rendering issue.

Reply:
seems to have fixed it at least in my local build, thanks!